scripts/utils/kill-port.ts

```diff
@@ -0,0 +1,67 @@
+// eslint-disable-next-line depend/ban-dependencies
+import { execa } from 'execa';
+
+// 99% inspired by https://github.com/tiaanduplessis/kill-port/blob/master/index.js
```

Comment:
If we want to follow the MIT licensing properly, we should also add `Copyright (c) Tiaan du Plessis`.

Reply:
good one! done